app/less/common.less

```diff
@@ -273,3 +273,7 @@ div[block-registration], div[block-editor] {
     }
   }
 }
+
+rule-message-alignment{
```

Comment:
@jayakrishnanv Good catch. Fixed in https://github.com/CruGlobal/conf-registration-web/pull/507.